Occasional VolumeIT failure on 1.9 branch

I'm seeing this occasional failure from VolumeIT on the 1.9 branch. This most recent failure occurred as of commit b67d3f93eea742523a98af95ca3a7aaceda5a990, but I've seen it earlier than that commit, so it's not new. I'm putting the specific commit down so the line numbers are easier to track down.

#### Error Message
```
Unexpected volume file:/var/lib/jenkins/workspace/1.9-Hadoop2/test/target/mini-tests/org.apache.accumulo.test.VolumeIT_testRemoveVolumes/volumes/v1/tables/1/t-000006k/A00000k9.rf
```

#### Stacktrace
```
java.lang.AssertionError: Unexpected volume file:/var/lib/jenkins/workspace/1.9-Hadoop2/test/target/mini-tests/org.apache.accumulo.test.VolumeIT_testRemoveVolumes/volumes/v1/tables/1/t-000006k/A00000k9.rf
	at org.junit.Assert.fail(Assert.java:88)
	at org.apache.accumulo.test.VolumeIT.verifyVolumesUsed(VolumeIT.java:445)
	at org.apache.accumulo.test.VolumeIT.testRemoveVolumes(VolumeIT.java:515)
	at java.base/jdk.internal.reflect.NativeMethodAccessorImpl.invoke0(Native Method)
	at java.base/jdk.internal.reflect.NativeMethodAccessorImpl.invoke(NativeMethodAccessorImpl.java:62)
	at java.base/jdk.internal.reflect.DelegatingMethodAccessorImpl.invoke(DelegatingMethodAccessorImpl.java:43)
	at java.base/java.lang.reflect.Method.invoke(Method.java:566)
	at org.junit.runners.model.FrameworkMethod$1.runReflectiveCall(FrameworkMethod.java:50)
	at org.junit.internal.runners.model.ReflectiveCallable.run(ReflectiveCallable.java:12)
	at org.junit.runners.model.FrameworkMethod.invokeExplosively(FrameworkMethod.java:47)
	at org.junit.internal.runners.statements.InvokeMethod.evaluate(InvokeMethod.java:17)
	at org.junit.internal.runners.statements.RunBefores.evaluate(RunBefores.java:26)
	at org.junit.internal.runners.statements.RunAfters.evaluate(RunAfters.java:27)
	at org.junit.internal.runners.statements.FailOnTimeout$CallableStatement.call(FailOnTimeout.java:298)
	at org.junit.internal.runners.statements.FailOnTimeout$CallableStatement.call(FailOnTimeout.java:292)
	at java.base/java.util.concurrent.FutureTask.run(FutureTask.java:264)
	at java.base/java.lang.Thread.run(Thread.java:834)

```

--- a/server/base/src/main/java/org/apache/accumulo/server/fs/VolumeUtil.java
+++ b/server/base/src/main/java/org/apache/accumulo/server/fs/VolumeUtil.java
@@ -187,8 +187,7 @@ public class VolumeUtil {
       VolumeManager vm, KeyExtent extent, TabletFiles tabletFiles, boolean replicate)
       throws IOException {
     List<Pair<Path,Path>> replacements = ServerConstants.getVolumeReplacements();
-    if (replacements.isEmpty())
-      return tabletFiles;
+
     log.trace("Using volume replacements: " + replacements);
 
     List<LogEntry> logsToRemove = new ArrayList<>();

